File: programming-languages/CSharp/data-input-output.cs
using System;

class Program {
    static void Main() {
        Console.Write("Digite seu nome: ");
        string nome = Console.ReadLine();

        Console.Write("Digite sua idade: ");
        int idade = int.Parse(Console.ReadLine());

        Console.WriteLine($"Olá, {nome}! Você tem {idade} anos.");
    }
}

File: programming-languages/Java/datainputoutput.java
import java.util.Scanner;

public class datainputoutput {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite seu nome: ");
        String nome = scanner.nextLine();

        System.out.println("Olá, " + nome + "!");
    }
}